### PR TITLE
Defined findElement

### PR DESCRIPTION
**Pull request by ijlee2**
## Description

I updated the `analyze-project` step so that the codemod knows:

- If `...attributes` exists
- Where `...attributes` appears

In a separate pull request, I will update the codemod to fill out `Signature['Element']`.

### CODE REVIEW

File: tests/fixtures/ember-container-query-glint/input/src/components/products/product/image.hbs [hunk 1]
```diff
@@ -1,12 +1,13 @@
 {{#if this.isTestEnvironment}}
-  <div local-class="placeholder-image"></div>
+  <div local-class="placeholder-image" ...attributes></div>
```
Comment (ijlee2) [inline tests/fixtures/ember-container-query-glint/input/src/components/products/product/image.hbs:2]: This component covers an edge case of `Element`. There is dynamic code as well as more than 1 `...attributes`, so it's hard to guarantee correctness.

https://github.com/emberjs/rfcs/blob/master/text/0748-glimmer-component-signature.md#components-with-multiple-or-varying-elements

File: tests/fixtures/ember-container-query-glint/input/src/components/products/product/card.hbs [hunk 2]
```diff
@@ -3,6 +3,7 @@
   @tagName="article"
   data-test-product-card
   local-class="container"
+  ...attributes
```
Comment (ijlee2) [inline tests/fixtures/ember-container-query-glint/input/src/components/products/product/card.hbs:6]: This component covers an edge case of `Element`. The splattributes are passed to a component, which happens to apply them on an `<article>` element. It's hard to guarantee correctness.
